### PR TITLE
Upgrade development to 0.2.5

### PR DESCRIPTION
**Pull request by g-krisztian**

### CODE REVIEW

**Comment by baskeboler** [comment]
is this related to #101 ?

**Comment by baskeboler** [comment]
I may be wrong, but this PR only includes formatting changes.

**Comment by soulflyer** [comment]
There are non whitespace changes in interceptor/queue.clj and webserver/core.clj, plus components is removed from the requires and framework and reitit are updated in project.clj.
I like the enforcing of cljstyle rules, but it should probably have been in another PR :-)

**Review by soulflyer — APPROVED** [review]
Looks good.

**Comment by baskeboler** [comment]
Can we remove the formatting changes from this PR? It makes finding the code changes like looking for a needle in a haystack. I agree with @soulflyer that we could do another PR with all the formatting updates.

**Comment by g-krisztian** [comment]
> Can we remove the formatting changes from this PR? It makes finding the code changes like looking for a needle in a haystack. I agree with @soulflyer that we could do another PR with all the formatting updates.

If you want, I can create another PR for formatting the already existing development and main

**Comment by baskeboler** [comment]
sure, if it makes the other code reviews more readable and less time consuming then I think it is a good idea.

**Comment by g-krisztian** [comment]
> sure, if it makes the other code reviews more readable and less time consuming then I think it is a good idea.

https://github.com/Flexiana/framework/pull/103

File: project.clj [hunk 1]
```diff
@@ -1,9 +1,8 @@
-(defproject com.flexiana/framework "0.2.4"
+(defproject com.flexiana/framework "0.2.5"
```
Comment (baskeboler) [inline project.clj:1]: please change version to 0.3.0

**Pull request closed without merging**